constants/theme.js

```diff
@@ -30,4 +30,5 @@ export const THEME = {
 export const fontSize = {
   12: '0.75rem',
   14: '0.875rem',
+  16: '1rem',
```

Comment:
did typography.fontSize not work when using 16px as the font size..?

Reply:
not sure what you mean by `typography.fontSize`. This is being used in a `TableCell` style.

Reply:
the sx prop is using the theme and I believe that the typography.fontSize is the default 16px/1rem. 
so 
```javascript
sx={{fontSize: 'typography.fontSize'}}
```

Reply:
oh I see. I think it's a little different, but you seem to be able to do this - https://github.com/AMP-SCZ/dpdash/pull/479/commits/16b3f5de405652b1c7f1984a4499061a306f57dc